api/src/controllers/utils.ts

```diff
@@ -163,6 +163,24 @@ router.post(
 	respond
 );
 
+router.post(
+	'/transform-export',
+	asyncHandler(async (req, res, next) => {
+		if (req.is('application/json') === false || !req.headers['content-type']) {
+			throw new InvalidPayloadException(`Payload should be a JSON object`);
+		}
+
+		if (!req.sanitizedQuery.export) {
+			throw new InvalidQueryException('"export" has to be provided.');
+		}
+
+		res.locals.payload = { data: req.body };
+
+		return next();
+	}),
+	respond
+);
```

Comment:
This whole endpoint is a no-op now, as the transform happens based on the ?export flag. I get why that's the case, but it feels a little.. odd. 

Also should we maybe call it `/utils/transform` (drop the `-export`) and allow the user to pass a `?format` flag similar to `?export` 🤔 

I'm probably overthinking this 🤷🏻